Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/SettingsAdapter.java

```diff
@@ -232,6 +232,11 @@ public void onSliderClick(SliderSetting item, int position)
     builder.setTitle(item.getNameId());
     builder.setView(view);
     builder.setPositiveButton(R.string.ok, this);
+    builder.setNeutralButton(R.string.default_value, (dialogInterface, i) ->
+    {
+      // Prevent button from closing the dialog to allow ease of manipulating multiple sliders.
+      // Overridden below.
```

Comment:
The connection between the two sentences isn't especially clear. What do you think about: "Overridden below to prevent button from closing the dialog." (Feel free to re-add something about ease of manipulating multiple sliders to what I wrote.)

Reply:
I like it.